data/db_demo.yaml

```diff
@@ -273,6 +294,20 @@ sources:
     group_ids:
       - =public_group_id
       - =program_A
+  - id: ZTFe028h94k
```

Comment:
are these real names ?

Reply:
Yes.  They are the names on the ZTF variable marshal.

Reply:
(true, but they are auto-generated and random)